tslint.json

```diff
@@ -29,7 +29,7 @@
     "no-string-literal": true,
```

Comment:
I'll do that next. Webpack has some pre-loader options. If that doesn't work then I'll just setup linting through CLI or gulp as a part of the watch script.